src/QueryDataTable.php

```diff
@@ -45,6 +45,11 @@ class QueryDataTable extends DataTableAbstract
      */
     protected $limitCallback;
 
+    public static function canCreate($source)
```

Comment:
Missing docs. Maybe we can use `@inheritdoc`? But I usually copy and paste the comments :)